src/plots/cartesian/select.js

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+var polygon = require('../../lib/polygon');
```

Comment:
@alexcjohnson a couple (still un-documented) :cow2: 

- `'use strict';` should have a blank line below it
- top-level `require` statements should be in groups separated by blank line in order: 
  1. node builtin modules
  2. npm public modules
  3. project files
  4. files within the same directory 
  5. other variable definitions

So, the `select.js` require blocks would be:

```js
'use strict';

var polygon = require('../../lib/polygon');

var axes = require('./axes');

var filteredPolygon = polygon.filter;
var polygonTester = polygon.tester;
var BENDPX = 1.5; // max pixels off straight before a line counts as bent
```